packages/keyv-mongo/src/index.ts

```diff
@@ -0,0 +1,112 @@
+import EventEmitter from 'events';
+import { MongoClient, Collection } from 'mongodb';
```

Comment:
`mongojs` was swapped for the native client which support promises ootb and has type definitions readily available

Reply:
That's fine, as long as the test suit still passes.